contributors/index.js

```diff
@@ -258,6 +258,15 @@ module.exports.list = [
     github_username: "Icor101",
     twitter_username: "#"
   },
+  
+  {
+    name: "Vishal",
+    country: "India",
+    role: "To be filled",
```

Comment:
Hi @Vishalyadav0804  .Can you fill in the `'role'` property with what type of developer you are or want to be !